instrumentation/logback/logback-appender-1.0/library/src/main/java/io/opentelemetry/instrumentation/logback/appender/v1_0/OpenTelemetryAppender.java

```diff
@@ -204,7 +204,9 @@ public void setOpenTelemetry(OpenTelemetry openTelemetry) {
     try {
       // minimize scope of write lock
       this.openTelemetry = openTelemetry;
-      this.eventsToReplay.drainTo(eventsToReplay);
+      if (openTelemetry != null) {
```

Comment:
Could you please explain the use case in which `setOpenTelemetry(null)` would be necessary?

Reply:
Our tests call `OpenTelemetryAppender.install(null)`, had to fix this because it produced a NPE.

Reply:
Perhaps add a comment to be able to know that it's for a testing purpose and not a specific user use case?